lib/tarantool_server.py

```diff
@@ -454,7 +454,8 @@ def seek_wait(self, msg, proc=None, name=None):
         with open(self.path, 'r') as f:
             f.seek(self.log_begin, os.SEEK_SET)
             cur_pos = self.log_begin
-            while True:
+            current_time = time.time()
```

Comment:
I would name it `start_time`, current naming is consufing. Or, as it is often written:

```python
deadline = time.time() + timeout
while time.time() < deadline:
    # <..workload..>
```

Reply:
You're right. Thank you!